docs/conf.py

```diff
@@ -124,7 +129,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
Why?  Does `tox -e docs` barf for you because of git's absurd ignore-empty-directories bug^Wdesign choice?